navbar.js

```diff
@@ -0,0 +1,38 @@
+var categoriesSort = categories.sort((a, b) => {
+  return b.count - a.count;
+});
+
+var tmp = 0;
+
+var firstFiveCategory = categoriesSort.filter(c => {
```

Comment:
witam w projekcie z przeszłości :D 
pracując z webpackerem i reactem mamy dostępny tzw. Babel, czyli silnik tłumaczący ES6 na ES5 rozumiane przez wszystkie przeglądarki (ES to upraszczając wersja javascriptu, poczytaj sobie proszę o tym, np. tu http://es6-features.org/ ale też generalnie o ECMAScripcie i jak to wszystko się odbywa w wolniejszej chwili )

tutaj wszędzie w projekcie używasz `arrow functions`, które nie jest rozumiane przez wszystkie przeglądarki (choć przez większość) - https://caniuse.com/#search=arrow
niestety jako profesjonaliści musimy sprawić aby działało zawsze i wszędzie.
Babela nie zainstalujemy do takiego projektu, dlatego niestety musimy przejść na staropolskie funkcje anonimowe. w tym przypadku ten sam kod będzie w ES5 wyglądał tak:
```
var firstFiveCategory = categoriesSort.filter(function(c) {
  tmp++;
  return tmp <= 5;
});
```
czyli generalnie niewielka zmiana, ALE haczyk jest taki, że zmienia się kontekst wywołania funkcji, dlatego użyte wewnątrz `this` (gdyby jakieś było) wskazywałoby na zupełnie inny obiekt niz w przypdku arrow function.
niestety zmiany musimy zrobić w całym projekcie

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array tutaj są podane metody, których można używać na Arrayach bez potrzeby dodawania lodasha czy innych bibliotek służących do takich operacji :) zawsze wybieramy natywne metody danego typu dla ES5, dopiero potem szukamy w lodashu i innych - taka notka na przyszłość